test/cypress/integration/25-host.apply.test.js

```diff
@@ -18,8 +18,6 @@ describe('apply to host', () => {
     cy.get('button[type="submit"]').click();
     cy.wait(1000);
     cy.get('[data-cy="collective-title"]', { timeout: 10000 }).contains('New collective');
-    cy.get('.NotificationBar h1').contains('success');
-    cy.get('.NotificationBar p').contains('BrusselsTogether ASBL');
```

Comment:
Why removing that?

Reply:
Once Modal is enabled, Notification bar is not there 

It's either Modal or Notification bar : https://github.com/opencollective/opencollective-frontend/blob/5376b7543a0c97f8689ad47792273cd35a772e37/components/create-collective/index.js#L83